app/assets/v2/js/pages/bounty_details.js

```diff
@@ -1240,6 +1240,9 @@ const build_uri_for_pull_bounty_from_api = function() {
   if (typeof document.issue_stdbounties_id != 'undefined') {
     uri = uri + '&standard_bounties_id=' + document.issue_stdbounties_id;
   }
+  if (typeof document.eventTag != 'undefined') {
+    uri = uri + '&event_tag=' + document.eventTag;
```

Comment:
can we use [concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/concat#Performance) aka
 `uri = uri.concat('&event_tag=', document.eventTag)` ?

Reply:
But the doc you sent me is saying to use assignment operators ( + ) instead of concat() 
"the assignment operators are several times faster"

Reply:
oh derp derp :P i misread that